.github/.wordlist.txt

```diff
@@ -1276,6 +1277,7 @@ sysconfdir
 SysConfig
 sysctl
 sysdeps
+Sysdiagnose
```

Comment:
Should we maybe use backticks to escape this ?

Reply:
Done